enhancements/installer/openstack-baremetal-compute.md

```diff
@@ -0,0 +1,193 @@
+---
+title: openstack-baremetal-compute
+authors:
+  - "@pierreprinetti"
+reviewers:
+  - "@openshift/team-openstack-cloud-provider"
+approvers:
+  - TBD
+creation-date: 2019-12-10
+last-updated: 2019-12-10
+status: provisional
+
+---
+
+# Openstack Baremetal Compute
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+OpenStack's [Nova][openstack-nova] service can be configured to provision
+baremetal machines as OpenShift Compute Nodes.
+
+This enhancement will allow [Ironic][openstack-ironic]-provisioned machines to
+be used as OpenShift Compute Nodes, when they are made available as Nova
+flavors.
+
+## Motivation
+
+In heavy-loaded clusters, or when the workload is better handled with specially
+purposed hardware, the functional requirement of controlling where the work is
+computed might outweight the added value of virtualisation.
+
+In OpenStack clusters where Nova transparently provisions baremetal machines,
+this enhancement will ensure that OpenShift installations can use them as
+Compute Nodes.
+
+### Goals
+
+* Enable running OpenShift on a mix of virtual and baremetal machines.
+* Baremetal machines can be added and removed from a running OpenShift cluster.
```

Comment:
It would be good to be slightly more verbose here, because when the world baremetal is used, it may be confused with the metal3.io work.